ci/setup_thrift.ps1

```diff
@@ -0,0 +1,15 @@
+$ErrorActionPreference = "Stop"
+trap { $host.SetShouldExit(1) }
+
+# TODO: consider moving this to .gitmodules
+git clone https://github.com/Microsoft/vcpkg.git
+Push-Location -Path vcpkg
+$VCPKG_DIR=(Get-Item -Path ".\").FullName
+setx VCPKG_DIR (Get-Item -Path ".\").FullName
+./bootstrap-vcpkg.bat
+./vcpkg integrate install
```

Comment:
Is this required for the build? This changes some global MSBuild settings which may interrupts existing vcpkg installation if any?

Reply:
Thanks yeah integrate install is not needed. I cleaned.